packages/graphback-runtime-mongodb/src/MongoDBDataProvider.ts

```diff
@@ -22,12 +24,12 @@ export class MongoDBDataProvider<Type = any, GraphbackContext = any> implements
     this.tableMap = buildModelTableMap(baseType);
     this.collectionName = this.tableMap.tableName;
     this.fieldTransformMap = getFieldTransformations(baseType);
+
+    findAndCreateIndexes(baseType, this.db.collection(this.collectionName)).catch((e: Error) => {
```

Comment:
Seems like the `applyIndexes` function already catches the error and does not re-thrown: https://github.com/aerogear/graphback/pull/1276/files#diff-0310a0356ee20ce16b7b5d3f6d8d6420R16, maybe this catch  clause here is redudant.

Reply:
I agree, but there's [this](https://palantir.github.io//tslint/rules/no-floating-promises/) rule, I will add a tslintignore comment in sometime, wdyt

Reply:
Okay, we can look at that later on. It was a minor nitpick.